HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -84,6 +84,17 @@ def customiseFor16569(process):
 
     return process
 
+def customiseFor17094(process):
+    for mod in ['hltHbhereco','hltHbherecoMethod2L1EGSeeded','hltHbherecoMethod2L1EGUnseeded','hltHfreco','hltHoreco']:
+        if hasattr(process,mod):
+            getattr(process,mod).timeSigmaSiPM = cms.double(2.5)
+            getattr(process,mod).pedSigmaSiPM = cms.double(0.00065)
+            getattr(process,mod).noiseSiPM = cms.double(1)
+            getattr(process,mod).ts4Max = cms.vdouble(100.,45000.)
+            getattr(process,mod).ts4chi2 = cms.vdouble(15.,15.)
+
+    return process
+
```

Comment:
ciao
just as a reminder, 
we should make sure this customization is run *only* in 2017 workflows,
because I think we do not want to modify the behaviour of the HLT menu for 2016 workflows